core/src/main/java/org/apache/carbondata/core/writer/ThriftWriter.java

```diff
@@ -80,6 +80,17 @@ public void open() throws IOException {
   }
 
   /**
+   * This will check whether stream and binary out is open or not.
+   * @return
+   */
+  public boolean isOpen() {
+    if (null != binaryOut && null != dataOutputStream) {
```

Comment:
in close  we are using util to close the stream so i think no need to set null explicitly.